### PR TITLE
pricing components added

### PR DESCRIPTION
**Pull request by Donfreddy**

### CODE REVIEW

**Comment by khatabwedaa** [comment]
Hi @Donfreddy, Thanks for adding a component to Meraki UI. is this component design by you.

File: components/Main.vue [hunk 1]
```diff
@@ -89,6 +112,8 @@ import SectionsAboutMe from "./ui/Sections/AboutMe";
 import SectionsFeature from "./ui/Sections/Feature";
 import SectionsOurTeam from "./ui/Sections/OurTeam";
 import SectionsParagraphWithImage from "./ui/Sections/ParagraphWithImage";
+// Pricing
+import PricingsSimple from "./ui//Pricings/Simple";
```
Comment (khatabwedaa) [inline components/Main.vue:116]: Pls, rename the new section to pricing instead of pricings

File: components/Main.vue [hunk 2]
```diff
@@ -157,9 +183,10 @@ export default {
     data() {
         return {
             categories: [],
-            searchText: 'Alerts',
+            // searchText: "Alerts",
+            searchText: "Pricings",
```
Comment (khatabwedaa) [inline components/Main.vue:187]: Back a search text to alerts because this will be in production after its merge.

Comment (Donfreddy) [reply components/Main.vue:187]: yes sorry

File: components/Main.vue (hunk 1 above)
Comment (Donfreddy) [reply components/Main.vue:116]: ok

**Comment by Donfreddy** [comment]
> Hi @Donfreddy, Thanks for adding a component to Meraki UI. is this component design by you.

No is design by Mubin ul haq Vhora and Amir Vhora on behance.
link: https://www.behance.net/gallery/29345451/WebWire-Web-Wireframe-UI-Stater-Kit

**Pull request closed without merging**